### DISCUSSION TITLE
Organize panels in columns

### DISCUSSION THREAD

**Discussion by juhotaipale** [laravel/nova-issues#4325] [Ideas & Feature Requests]
As a feature request, it would be nice if we can organize panels in columns in detail view.

Sometimes it saves space and also is visually more clean with large screen if we could have data panels organized next to each other.

It may be something like `(new Column)->width(1/2)`

Thoughts? Or can we already implement this somehow? It seems that the container doesn't have flexbox out of the box.

**Comment by ianrobertsFF** [comment]
I've achieved something similar with [CSS Grid](https://css-tricks.com/snippets/css/complete-guide-grid/) and the theming classes in both Nova 3 and Nova 4 now

Theming classes are built in to Nova 3 and you can follow the docs to activate, or for Nova 4 I have a package to add them https://github.com/Formfeed-UK/nova-theming-classes
